students/14th/team9/youngjoo/project_westargram/account/views.py

```diff
@@ -0,0 +1,88 @@
+import json
+import re
+from django.shortcuts import render
+from django.views import View
+from django.http import JsonResponse
+from .models import Accounts
+
+
+class AccountsView(View):
+#필수 정보( 전화번호, 이름, 이메일, 비밀번호)
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+# 이메일 형식 
+            p = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+            if p.match(data['email']) == None:
```

Comment:
```suggestion
            if not p.match(data['email']):
```
`None` 은 Falsy value 이므로 이런식으로 변경해줄 수 있습니다.
조금 더 알고싶으시다면 파이썬에서 Truthy, Falsy value 에 대해 검색해보세요.